searx/settings.yml

```diff
@@ -1190,6 +1190,20 @@ engines:
       require_api_key: false
       results: JSON
 
+  # - name: mullvadleta
+  #   engine: mullvad_leta
+  #   use_cache: true  # Only 100 non-cache searches per day, suggested only for private instances
+  #   search_url: https://leta.mullvad.net
+  #   categories: [general, web]
+  #   shortcut: ml
+  #   about:
+  #     website: https://leta.mullvad.net/
+  #     wikidata_id: Q47008412
+  #     official_api_documentation: https://leta.mullvad.net/faq
+  #     use_official_api: false
+  #     require_api_key: false
```

Comment:
Its already in mullvad_leta.py

Reply:
Can you please have a look and remove duplicates already given `searx/engines/mullvad_leta.py`